src/modules/readout/vmm3/CMakeLists.txt

```diff
@@ -14,3 +14,13 @@ set(VMM3ParserTest_SRC
   VMM3Parser.cpp
 )
 create_test_executable(VMM3ParserTest)
+
+
+set(VMM3CalibrationTest_INC
+  VMM3Calibration.h
+)
+set(VMM3CalibrationTest_SRC
+  test/VMM3CalibrationTest.cpp
+  VMM3Calibration.cpp
```

Comment:
Is it intentional that this is an empty file?

Reply:
Nah, just the original intention that there was both a header and implementation. Turned out the implementation could fit in the header. I'll remove that.